src/utils/transaction.ts

```diff
@@ -0,0 +1,38 @@
+import { ParsedStruct } from '../contract';
+import { Call } from '../types';
+import { getSelectorFromName } from './hash';
+import { BigNumberish, bigNumberishArrayToDecimalStringArray, toBN } from './number';
+
+export const fromCallsToCallArray = (calls: Call[]) => {
```

Comment:
I would suggest adding some JSDoc to the function what it does and why it's there. And maybe renaming the function.

`transformCallsToMultiCallsArray()` might be more descriptive